contracts/Config.sol

```diff
@@ -9,7 +9,7 @@ contract Config is IConfig, OwnableUpgradeable {
 
     address private crankAddress;
     address private feeReceiver;
```

Comment:
this should be public--no need for the function for returning this value

Reply:
updated